lib/stormpath-sdk/id_site/error.rb

```diff
@@ -0,0 +1,42 @@
+module Stormpath
+  module IdSite
+    class Error < Stormpath::Error
```

Comment:
This class is used to create errors based on validations on the client side, if I understand correctly. I don't think this is something that we should be doing on the SDK since the server is supposed to take care of these validations and return the corresponding status code errors; otherwise we would be duplicating the logic that is already in place server side. Also, we shouldn't be optimizing for status code errors because something could change on the server and that will no longer be in sync with the SDK.

Let me know if I'm missing something.